tests/integration/pydecred/test_rpc.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+import os
+from tinydecred.pydecred import rpc
+from tinydecred.util import helpers
+
+
+@pytest.fixture
+def config():
+    dcrdCfgDir = helpers.appDataDir("dcrd")
+    cfgPath = os.path.join(dcrdCfgDir, "dcrd.conf")
+    if not os.path.isfile(cfgPath):
+        return None
+    cfg = helpers.readINI(cfgPath, ["rpcuser", "rpcpass", "rpccert"])
+    assert "rpcuser" in cfg
+    assert "rpcpass" in cfg
+    if "rpccert" not in cfg:
+        cfg["rpccert"] = os.path.join(dcrdCfgDir, "rpc.cert")
+    if "rpclisten" not in cfg:
+        cfg["rpclisten"] = "localhost:9109"
```

Comment:
Could you check if testnet is specified and use testnet's default port if so?

Reply:
I'd rather force testing on mainnet. That way we can test against on-chain data. Testnet will eventually be changing.

Reply:
hmmm. My mind blown.